src/Integration/ETW/CodeMarkers.cs

```diff
@@ -87,6 +89,40 @@ public static class Keywords
 
         #endregion
 
+        #region Analysis: 2000-2999
```

Comment:
From readability point of view, it might help to rename this to "General analysis" and the other to "CFamily analysis"

Reply:
`IMessageHandler` is actually CFamily-specific (I originally thought it was generic). So we don't have any generic analysis events at the moment.

I'll move the HandleMessage events to the `CFamily` section (which will leave `Analysis` range empty, so I'll remove it).